packages/frontend-web/src/app/scenes/Settings/sections/RestrictedTerms.tsx

```diff
@@ -0,0 +1,149 @@
+import { ChangeEvent, Component } from "react";
+
+import AddRestrictedTerm, { RestrictedTermLevels } from "./AddRestrictedTerm";
+import { AddButton } from "../components/AddButton";
+
+import { globalRestrictedTerms } from "../../../platform/restrictedTermsService";
+import { IRestrictedTermAttributes } from "../../../../models";
+
+import { css } from "../../../utilx";
+import { SETTINGS_STYLES } from "../settingsStyles";
+import { autobind } from "core-decorators";
+
+export interface ISectionProps {
+  styles: any;
+  settingsState: any;
+  setSettingsState: any;
+}
+
+export interface IRestrictedTermsState {
+  terms: IRestrictedTermAttributes[];
+  displayAddTerm: boolean;
+  disableMenu: boolean;
+}
+
+export class RestrictedTerms extends Component<ISectionProps, IRestrictedTermsState> {
+  state: IRestrictedTermsState = {
+    terms: [],
+    displayAddTerm: false,
+    disableMenu: false,
+  };
+
+  @autobind
+  toggleDisableMenu() {
+    this.setState({ disableMenu: !this.state.disableMenu });
+  }
+
+  @autobind
+  async getTerms() {
+    // get rid of Try/Catch when done with dev
+    try {
+      const terms = await globalRestrictedTerms.get();
+      this.setState({ terms: terms });
+    } catch (err) {
+      console.log("error getting terms", err);
+    }
+  }
+
+  @autobind
+  async deleteTerm(termId: string) {
+    try {
+      this.toggleDisableMenu();
+      await globalRestrictedTerms.delete(termId);
+      await this.getTerms();
+    } catch (err) {
+      console.log("error occurred deleting a term", err);
+    }
+    this.toggleDisableMenu();
+  }
+
+  @autobind
+  async updateTermScore(event: ChangeEvent<HTMLSelectElement>, term: IRestrictedTermAttributes) {
+    this.toggleDisableMenu();
+    const updatedTerm = {
+      ...term,
+      score: parseFloat(event.target.value),
+    };
+    try {
+      await globalRestrictedTerms.update(updatedTerm);
+      this.getTerms();
+    } catch (error) {
+      console.log("error occurred updating a term", error);
+    }
+    this.toggleDisableMenu();
+  }
+
+  @autobind
+  toggleDisplayAddTerm() {
+    this.setState({ displayAddTerm: !this.state.displayAddTerm });
+  }
+
+  componentDidMount() {
+    this.getTerms();
+  }
+
+  render() {
+    const { styles } = this.props;
+
+    return (
+      // Almost more jaring with the menu disabled. Not sure what is best to do.
+      <div key="restrictedTermsSection" {...css(this.state.disableMenu && SETTINGS_STYLES.disableMenu)}>
+        <div key="heading" {...css(styles.heading)}>
+          <h2 {...css(styles.headingText)}>Restricted Terms</h2>
+        </div>
+        <div key="body" {...css(styles.section)}>
+          {/* Temporary wording. May need updating */}
+          <p>These terms are restricted and will cause a comment to be flagged.</p>
```

Comment:
Maybe something more along the lines of 

`Comments containing these terms will receive Restricted Term tags with scoring corresponding to their assigned level`